schemata/profile-list-result-schema.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "profileListResult",
+  "description": "A profileListResult object is a result object with additional fields to indicate the result of processing a profile list message.",
+  "type": "object",
+  "properties": {
+    "status": {
+      "description": "Status code of the result",
+      "type": "string",
+      "enum": ["NEW", "SUCCESS", "WARNING", "FAILED", "MISSING", "DELETE"]
+    },
+    "message": {
+      "description": "Description of the result",
+      "type": "string"
+    },
+    "sourceField": {
+      "description": "If the incoming message was transformed, this identifies the field of the incoming message to which the result pertains",
+      "type": "string"
+    },
+    "sourceValue": {
+      "description": "If the incoming message was transformed, the original value in the incoming message to which the result pertains",
+      "type": "string"
+    },
+    "field": {
+      "description": "JSON pointer identifying the SHIP field to which the result pertains",
+      "type": "string"
+    },
+    "summary": {
+      "description": "Summary of the profile list result",
+      "$ref": "#/definitions/summary"
+    },
+    "profiles": {
+      "description": "Profile results produced by processing the incoming message",
+      "type": "array",
+      "items": {
+        "$ref": "profile-result-schema.json#"
+      }
+    },
+    "errors": {
+      "description": "Errors generated by the processing of the guest-stay list. Deprecated, use guestStays instead.",
+      "type": "array",
+      "items": {
+        "$ref": "result-schema.json#"
+      }
+    },
+    "warnings": {
+      "description": "Warnings generated by the processing of the guest-stay list. Deprecated, use guestStays instead.",
+      "type": "array",
+      "items": {
+        "$ref": "result-schema.json#"
+      }
+    }
+  },
+  "additionalProperties": false,
+  "definitions": {
+    "summary": {
```

Comment:
A little suboptimal to redeclare summary object here. Perhaps we can move the guestStayListResult summary into common objects and make it more general, and just reference it here?